deeplabcut/live/dlc_live.py

```diff
@@ -32,131 +30,180 @@ class DLCLive(object):
             self.cfg['x1'], self.cfg['x2'], self.cfg['y1'], self.cfg['y2'] = self.cropping
             print("Overwriting cropping parameters:", self.cropping)
 
-        self.cfg["project_path"] = os.path.dirname(config)
-        modelfolder = os.path.join(self.cfg["project_path"],str(auxiliaryfunctions.GetModelFolder(trainFraction, self.shuffle, self.cfg)))
-        path_test_config = Path(modelfolder) / 'test' / 'pose_cfg.yaml'
-        try:
-            self.dlc_cfg = load_config(str(path_test_config))
-        except FileNotFoundError:
-            raise FileNotFoundError("It seems the model for shuffle %s and trainFraction %s does not exist."%(self.shuffle, trainFraction))
-
         # Check which snapshots are available and sort them by # iterations
         try:
-          Snapshots = np.array([fn.split('.')[0]for fn in os.listdir(os.path.join(modelfolder , 'train'))if "index" in fn])
+            snapshots = np.array([fn.split('.')[0]for fn in os.listdir(os.path.join(self.model_folder , 'train'))if "index" in fn])
         except FileNotFoundError:
-          raise FileNotFoundError("Snapshots not found! It seems the dataset for shuffle %s has not been trained/does not exist.\n Please train it before using it to analyze videos.\n Use the function 'train_network' to train the network for shuffle %s."%(shuffle,shuffle))
+            raise FileNotFoundError("Snapshots not found! It seems the dataset for shuffle {} has not been trained/does not exist.".format(self.shuffle) +
+                                    "Please train it before using it to analyze videos." +
+                                    "Use the function 'train_network' to train the network for shuffle {}.".format(self.shuffle))
 
         if self.cfg['snapshotindex'] == 'all':
-            print("Snapshotindex is set to 'all' in the config.yaml file. Running video analysis with all snapshots is very costly! Use the function 'evaluate_network' to choose the best the snapshot. For now, changing snapshot index to -1!")
+            Warning("Snapshotindex is set to 'all' in the config.yaml file." +
+                    "Running video analysis with all snapshots is very costly!"+
+                    "Use the function 'evaluate_network' to choose the best the snapshot." +
+                    "For now, changing snapshot index to -1!")
             snapshotindex = -1
         else:
             snapshotindex = self.cfg['snapshotindex']
 
-        increasing_indices = np.argsort([int(m.split('-')[1]) for m in Snapshots])
-        Snapshots = Snapshots[increasing_indices]
+        # sort snapshots by number of training (intervals??? what u call those?)
+        increasing_indices = np.argsort([int(m.split('-')[1]) for m in snapshots])
+        snapshots = snapshots[increasing_indices]
 
-        print("Using %s" % Snapshots[snapshotindex], "for model", modelfolder)
+        print("Using {} for model {}".format(snapshots[snapshotindex], self.model_folder))
```

Comment:
can this change be explained?

Reply:
% formatting was intended to be replaced by .format() and is bound for eventual deprecation
https://www.python.org/dev/peps/pep-3101/

Reply:
but also np to revert